ogre/src/OgreLightVisual.cc

```diff
@@ -75,7 +75,9 @@ void OgreLightVisual::CreateVisual()
     this->dataPtr->line = std::shared_ptr<OgreDynamicLines>(
       new OgreDynamicLines(MT_LINE_LIST));
     this->ogreNode->attachObject(this->OgreObject());
-    this->dataPtr->line->setMaterial("Default/TransGreen");
+    MaterialPtr lightVisualMaterial =
+      this->Scene()->Material("Default/TransGreen")->Clone();
+    this->SetMaterial(lightVisualMaterial, false);
```

Comment:
is this needed? since we're not modifying the material, we don't need a copy right?

Reply:
Fixed https://github.com/ignitionrobotics/ign-rendering/pull/228/commits/e8ea4e72e0d0ad7b9cf7c6e9d951e7f3b0c240b5